leader-election-impl/src/main/java/com/palantir/paxos/PaxosQuorumChecker.java

```diff
@@ -183,9 +193,7 @@ private PaxosQuorumChecker() {
 
         } finally {
             // cancel pending futures (during failures)
```

Comment:
nit: probably can get rid of `(during failures)` here - the original comment is not quite right